scalahost/nsc/src/main/scala/scala/meta/internal/semantic/AttributesOps.scala

```diff
@@ -264,13 +265,15 @@ trait AttributesOps { self: DatabaseOps =>
               gtree match {
                 case gview: g.ApplyImplicitView =>
                   val pos = gtree.pos.toMeta
-                  val syntax = gview.fun + "(*)"
+                  val syntax = g.showCode(gview.fun) + "(*)"
```

Comment:
What's the difference between `toString` and `showCode` here? I'm very wary of `showCode`, because it's known to sometimes swallow snippets of code that it considers irrelevant.